klass-api/src/main/resources/application.properties

```diff
@@ -46,6 +44,8 @@ management.metrics.tags.application=klass
 # solr Properties
 klass.env.search.solr.url=http://localhost:8983/solr
 klass.env.search.solr.core=Klass
+spring.elasticsearch.uris=http://localhost:9200
```

Comment:
What is the relationship between this and `opensearch.url`? Are they both necessary?